blocks/product-list-page-custom/product-list-page-custom.js

```diff
@@ -304,6 +328,7 @@ class ProductListPage extends Component {
       category: {
         name: headline,
         id: category || null,
```

Comment:
do we still need category id too?

Reply:
Unfortunately yes. The PLP does a `categories` query which requires the id, since we cannot filter by `urlPath` there (https://github.com/hlxsites/aem-boilerplate-commerce/blob/bulk-improvements/blocks/product-list-page-custom/product-list-page-custom.js#L24-L28).